package.json

```
@@ -2821,6 +2821,131 @@
+													{
+														"key": "g",
+														"name": "Peek definition",
+														"type": "command",
+														"command": "editor.action.peekDefinition"
+													},
+													{
+														"key": "d",
+														"name": "Peek declaration",
+														"type": "command",
+														"command": "editor.action.peekDeclaration"
+													},
+													{
+														"key": "r",
+														"name": "Peek references",
+														"type": "command",
+														"command": "editor.action.referencesSearch.trigger"
+													},
+													{
+														"key": "s",
+														"name": "Go to symbol in editor",
+														"type": "command",
+														"command": "workbench.action.gotoSymbol"
+													},
+													{
+														"key": "a",
+														"name": "Switch Header/Source",
+														"type": "command",
+														"command": "C_Cpp.SwitchHeaderSource"
+													},
+													{
+														"key": "e",
+														"name": "Go to errors/problems",
+														"type": "command",
+														"command": "workbench.actions.view.problems"
+													},
+													{
+														"key": "f",
+														"name": "Go to file in explorer",
+														"type": "command",
+														"command": "workbench.files.action.showActiveFileInExplorer"
+													},
+													{
+														"key": "G",
+														"name": "Go to definition",
+														"type": "command",
+														"command": "editor.action.revealDefinition"
+													},
+													{
+														"key": "D",
+														"name": "Go to declaration",
+														"type": "command",
+														"command": "editor.action.revealDeclaration"
+													},
+													{
+														"key": "R",
+														"name": "Find all references",
+														"type": "command",
+														"command": "references-view.findReferences"
+													}
+												]
+											},
+											{
+												"key": "r",
+												"name": "+Refactor",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "s",
+														"name": "Rename Symbol",
+														"type": "command",
+														"command": "editor.action.rename"
+													}
+												]
+											},
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format Document",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "s",
+														"name": "Format Selection",
+														"type": "command",
+														"command": "editor.action.formatSelection"
+													}
+												]
+											},
+											{
+												"key": "i",
+												"name": "+IntelliSense",
```

```suggestion
												"key": "b",
												"name": "+Backend",
```
https://develop.spacemacs.org/layers/+tools/lsp/README.html
Should we use `b` and "Backend" for this to be consistent? We use `b` for this purpose in other language's major mode as well.

---

Sure. If helps keep consistency across all layers I'd be happy to update it. Which two bindings should I updated it with.

---

Updated to use `b` for consistency.